storage/perfschema/pfs_digest.cc

```diff
@@ -75,7 +75,7 @@ int init_digest(const PFS_global_param *param)
   */
   digest_max= param->m_digest_sizing;
   digest_lost= 0;
-  PFS_atomic::store_u32(& digest_monotonic_index.m_u32, 1);
+  digest_monotonic_index.m_u32.store(1);
```

Comment:
> static PFS_ALIGNED PFS_cacheline_uint32 digest_monotonic_index;
why would someone make both alignas(cache line) and wrap it into an char array-aligned struct (`PFS_cacheline_uint32`)? @dr-m do you have any idea, or is it just an overingineering coming from the mysql port?